lib/sycamore/sycamore/query/planner.py

```diff
@@ -74,6 +74,307 @@
     "8. The last step of each plan *MUST* be a **SummarizeData** operation that returns a natural language response."
 )
 
+PLANNER_EXAMPLES = """
```

Comment:
can you convert this to a list? the immediate use case is using different formatting based on llm. I want xml <example> for claude